### PR TITLE
fix: map to render nav links for mobile navbar

### PR DESCRIPTION
**Pull request by adity2911**
## Description

Instead of writing individual links to render at the navbar, only updating the linksComponent will do so now.

### CODE REVIEW

**Comment by priyankarpal** [comment]
in future please avoid creating PR first. @adity2911

File: components/SideMenu.tsx [hunk 1]
```diff
@@ -19,6 +19,16 @@ function SideMenu(props: { handleDrawerToggle: () => void }) {
     },
   ];
 
+  const linksComponent = sideLinks.map((link, index) => (
+    <li>
```
Comment (priyankarpal) [inline components/SideMenu.tsx:23]: add the `key={index}` prop to the `<li>` element

``` suggestion
    <li key={index}>
```

File: components/SideMenu.tsx [hunk 2]
```diff
@@ -19,6 +19,16 @@ function SideMenu(props: { handleDrawerToggle: () => void }) {
...
+      <Link href={link.path} key={index} rel="noreferrer" className={`block py-2 px-3 w-fit m-2 text-left font-bold rounded-md`} aria-label="Github">
```
Comment (priyankarpal) [inline components/SideMenu.tsx:24]: remove `key={index}`

```suggestion
      <Link href={link.path}  rel="noreferrer" className={`block py-2 px-3 w-fit m-2 text-left font-bold rounded-md`} aria-label="Github">
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — CHANGES_REQUESTED** [review]
change the PR title to `fix: map to render nav links for mobile navbar`

**Comment by adity2911** [comment]
@priyankarpal The requests have been completed. Kindly let me know if you want anything else.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**